includes/attendee-repository.php

```diff
@@ -18,12 +16,20 @@ public function add_attendee( int $event_id, int $user_id ): void {
 			throw new Exception( 'invalid user id' );
 		}
 
-		$event_ids = get_user_meta( $user_id, self::USER_META_KEY, true );
-		if ( ! $event_ids ) {
-			$event_ids = array();
-		}
-		$event_ids[ $event_id ] = true;
-		update_user_meta( $user_id, self::USER_META_KEY, $event_ids );
+		global $wpdb, $gp_table_prefix;
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		$wpdb->query(
+			$wpdb->prepare(
+				"insert ignore into {$gp_table_prefix}event_attendees (event_id, user_id) values (%d, %d)",
```

Comment:
Why don't we use `$wpdb->insert()` here? Then we wouldn't need the disables from above.

Reply:
Since we're doing an `insert ignore`, I looked for a way to do insert ignore with `$wpdb->insert()` but it doesn't seem to be possible. Should I leave as is, or should we first delete and then `$wpdb->insert()`?

Reply:
Or we could through an exception if we're trying to insert an attendee that already exists.

Reply:
An `INSERT IGNORE` is just an insert that doesn't return an error if it couldn't insert a row because it already existed. So if we just ignore the return value then just using `$wpdb->insert()` should be fine?

Reply:
I tried it and indeed it works, though warning are written to the log, which also happens during tests, so it pollutes the test output. An example of these warnings:

> WordPress database error Duplicate entry '35-17' for key 'event_per_user'

Would it be ok to leave it as is?